Source/DafnyCore/AST/Grammar/ParseErrors.cs

```diff
@@ -116,18 +187,21 @@ public enum ErrorId {
     Add(ErrorId.p_literal_string_required,
     @"
 The value of an options attribute cannot be a computed expression. It must be a literal string.
-", Replace("\"\"", "replace with empty string"));
+", range => new List<DafnyAction> {
+    OneAction("remove " + range.PrintOriginal(), range, ""), // TODO - remove surrounding whitespace?
+    OneAction("replace with empty string", range, "\"\""),
```

Comment:
I would put only the third, that is, enclose in quotes.
Deleting can be done very easily but usually won't solve these kinds of problem, and I don't see why someone would be interested to remove what they wrote to put an empty string instead.

Reply:
OK